### ISSUE TITLE
Legacy rename

### ISSUE THREAD

**Issue by theriex** [theriex/membic#23]
For historical reasons, a membic is represented in the object model as a review, and a theme is represented as a coop.  Keeping extra names increases development impedence, but switching over the codebase and associated data looks like a lot of high risk work.

Keeping the db.Model declarations as legacy and transforming everything else is possible, but still less than optimal and it doesn't reduce the amount of work or risk by orders of magnitude.

Basically, this task is marking the need for a tranformation tool capable of finding the mentions, tracing the dependencies, tracing named variables, and basically providing all the capabilities to remove the risk from this process.

**Comment by theriex** [comment]
Turns out the transformation tool was me, porting the site to new hosting.  Source code is much better now having been largely rewritten.
